chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md

```diff
@@ -45,9 +45,10 @@ Multiple licenses can be represented using a SPDX license expression as defined
 
 Examples:
 
-    SPDX-License-Identifier: (GPL-2.0 OR MIT)
-    SPDX-License-Identifier: (LGPL-2.1 AND BSD-2-CLAUSE)
-    SPDX-License-Identifier: (GPL-2.0+ WITH Bison-exception-2.2)
+    SPDX-License-Identifier: (GPL-2.0-only OR MIT)
+    SPDX-License-Identifier: (LGPL-2.1-only AND BSD-2-CLAUSE)
+    SPDX-License-Identifier: (GPL-2.0-or-later[p;04ejr4\,er4\5j\ .] WITH Bison-exception-2.2)
```

Comment:
Did something get added incorrectly here following the `-or-later` addition? There seems to be an odd bracketed bit that I'm not sure what it's doing there...

Reply:
Me either, I blame my cat! I will get this fixed up.

Reply:
Fixed.